content/docs/pipelines/deployment-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Standalone Deployment Alternatives"
+description = "Alternative approaches to deploy Kubeflow Pipelines standalone to a cluster"
+weight = 15
++++
+
+As an alternative to [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize
+manifests.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+
+## Deploying Kubeflow Pipelines standalone to an existing cluster
+
+1. Deploy latest version of Kubeflow Pipelines:
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
+kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
```

Comment:
I'm consistently using github url to make manifests easier to understand and more discoverable.

Reply:
remove the double // in the path
we publish this to gcs now https://github.com/kubeflow/pipelines/blob/master/.release.cloudbuild.yaml#L209

Reply:
discussed, we will keep using github url here